tools/v8_gypfiles/v8.gyp

```diff
@@ -967,6 +976,21 @@
             ],
           },
         }],
+        ['OS=="zos"', {
+          'dependencies': [
+            '<!(echo $ZOSLIB_GYP):zoslib#target',
+          ],
+          'include_dirs+': [
+            '<!(echo $ZOSLIB_INCLUDES)',
```

Comment:
I think we can remove this if we are specifying includes via config.gypi

I believe only the `dependencies` triggers rebuild on modification.

Minor: We may also want to consider passing ZOSLIB_GYP as a gyp variable (via configure.py) instead of using an environment variable